pontoon/base/migrations/0069_notification_categories.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from django.db import migrations
+
+
+def get_category(notification):
+    verb = notification.verb
+    desc = notification.description
+
+    # New strings notifications
+    if re.match(r"updated with \d+ new string", verb):
+        return "new_string"
+
+    # Project target dates notifications
+    if re.match(r"due in \d+ days", verb):
+        return "project_deadline"
+
+    # Comments notifications
+    if re.match(r"has (pinned|added) a comment in", verb):
+        return "comment"
+
+    # New suggestions ready for review notifications
+    if verb == "":
+        return "unreviewed_suggestion"
+
+    if re.match(r"has reviewed suggestions", verb):
+        # Review actions on own suggestions notifications
+        if desc.startswith("Your suggestions have been reviewed"):
+            return "review"
+
+        # New team contributors notifications
+        if desc.startswith("<a href="):
+            return "new_contributor"
+
+
+def store_notification_categories(apps, schema_editor):
+    Notification = apps.get_model("notifications", "Notification")
+    notifications = Notification.objects.filter(data__isnull=True)
```

Comment:
Won't this leave all `direct_message` notifications unmigrated?

Reply:
Yeah. 

We don't need to categorize them for the sake of this PR, and I worry we'll override something in the data attributes. So I'd rather take care of them as part of https://github.com/mozilla/pontoon/issues/3464. Will add a note there.

Reply:
I changed my mind. 😎 

https://github.com/mozilla/pontoon/pull/3456/commits/928e52fb58374706c5bfcdf50a860e51061887b8